lib/dialects/postgres/index.js

```diff
@@ -62,34 +62,40 @@ class Client_PG extends Client {
     return `"${value.replace(/"/g, '""')}"${arrayAccessor}`;
   }
 
+  _acquireOnlyConnection() {
```

Comment:
Why was introduction of this method and modification of `acquireRawConnection` necessary if pgnative doesn't override either?

Reply:
This goes back to `connect` using the deprecated (at least according to docs) callback signature where the same connection is returned: https://github.com/brianc/node-postgres/blob/master/packages/pg/lib/client.js#L279

The native client doesn't do this. The mocking was dependent on this behavior, which is now undocumented, so I moved it toward being compliant with the current `node-postgres` documentation + compatible with `pg-native`

Reply:
https://github.com/brianc/node-postgres/blob/master/packages/pg/lib/native/client.js#L119-L134 native version for comparison.